CMakeLists.txt

```diff
@@ -188,6 +188,14 @@ if(UNIX AND APPLE)
     # Add the icon
     install(FILES ${Launcher_Branding_ICNS} DESTINATION ${RESOURCES_DEST_DIR} RENAME ${Launcher_Name}.icns)
 
+    # Add translation directories
+    file(GLOB TRANSLATIONS RELATIVE "${CMAKE_SOURCE_DIR}/Translations" "${CMAKE_SOURCE_DIR}/Translations/*.ts")
+    list(REMOVE_ITEM TRANSLATIONS ".template.ts")
+    list(APPEND TRANSLATIONS "en_US.ts")
+    list(TRANSFORM TRANSLATIONS REPLACE "\.ts" ".lproj")
+    list(TRANSFORM TRANSLATIONS PREPEND "${CMAKE_INSTALL_PREFIX}/${RESOURCES_DEST_DIR}/")
+    file(MAKE_DIRECTORY ${TRANSLATIONS})
```

Comment:
I wonder, is the bug fixed if you add every supported language to [`CFBundleLocalizations`](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-109552) in the `Info.plist` instead? It looks like it’s made specifically for these types of situations, and doesn’t require a bunch of empty directories.

> `CFBundleLocalizations` (`Array` - iOS, macOS) identifies the localizations handled manually by your app. If your executable is unbundled or does not use the existing bundle localization mechanism, you can include this key to specify the localizations your app does handle.

Reply:
I'm spread pretty thin at the moment, I think I'll find some time to look at it next week.

Reply:
I found some time to test it myself. I think `CFBundleLocalizations` works. I added the below to a release version's `Info.plist` (not comprehensive, obviously):

```
    <key>CFBundleLocalizations</key>
    <array>
        <string>en</string>
        <string>zh</string>
        <string>fr</string>
        <string>de</string>
        <string>it</string>
        <string>ja</string>
        <string>ko</string>
        <string>es</string>
    </array>
```

Then set my first system language to Spanish (`es`), moved my `~/Library/Application Support/PolyMC` folder to a different location (so it acts like a first-time startup), and it selected Spanish by default. No empty directories needed.